subsys/bluetooth/controller/ll_sw/ull_llcp_enc.c

```diff
@@ -406,6 +406,14 @@ static void lp_enc_st_wait_rx_enc_rsp(struct ll_conn *conn, struct proc_ctx *ctx
 		/* Pause Rx data */
 		ull_conn_pause_rx_data(conn);
 		lp_enc_store_s(conn, ctx, pdu);
+
+		/* After the Central receives the LL_ENC_RSP PDU in response,
+		 * only PDUs required by this procedure are expected;
+		 * to achieve that even "unwanted" PDU's end up in this FSM,
+		 * enable the greedy RX behaviour.
+		 */
```

Comment:
Suggestion for reformulation:
	/* After the Central has received the LL_ENC_RSP PDU,
		 * only PDUs related to this procedure are valid, and invalids should 
		 * result in disconnect.
		 * to achieve this enable the greedy RX behaviour, such that
		 * all PDU's end up in this FSM. 
		 */

Reply:
ACK